### DISCUSSION TITLE
The cursor is flicker when start cef process

### DISCUSSION THREAD

**Discussion by tungnt90** [cefsharp/CefSharp#4372] [General]
The cursor is flicker when start cef process on windows form application. How can i resolve this.

Thank in advance!

**Comment by amaitland** [comment]
> The cursor is flicker when start cef process on windows form application.

What do you mean by this?

You need to provide more information, see https://github.com/cefsharp/CefSharp/wiki/Questions-&-Support#when-asking-a-question-please-include-the-following

**Reply by tungnt90** [reply]
thank for your interesting. It look this

![cursor](https://user-images.githubusercontent.com/1974628/214037551-9411540d-7f67-4639-bf04-2de85cb37c8f.jpg)

when cefsharp starting or new page opened.

**Reply by amaitland** [reply]
Does the problem reproduce with https://github.com/cefsharp/CefSharp.MinimalExample

You need to answer the questions as per the link I've provided, there's not enough information here to even speculate.
